### PR TITLE
docs: update tasks section

### PR DESCRIPTION
**Pull request by TC-MO**
fix typos
rewrites for clarity and conciseness
fix naming of the UI elements

### CODE REVIEW

File: sources/platform/actors/running/tasks.md [hunk 1]
```diff
@@ -1,51 +1,45 @@
...
 
-Open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create empty task** button.
+To create a task, open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create task** button.
 
 ![Create a new Apify task](./images/tasks/create-task.png)
 
 ## Configure
 
-You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it is just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
+You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:24]: It's just Input now
```suggestion
You can set up your task's input under the **Input** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
```

File: sources/platform/actors/running/tasks.md [hunk 2]
```diff
@@ -1,51 +1,45 @@
...
 
 ### Settings
 
-You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of their detail page.
+You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of the task's detail page.
 
 ## Run
 
-Once you've configured you task, you can run it using the **Run** button in the bottom-left side of the **Input and options** tab.
+Once you've configured your task, you can run it using the **Run** button in the bottom-left side of the **Input and options** tab.
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:42]: I guess the move of the Run button is going to mess some things up for Docs

```suggestion
Once you've configured your task, you can run it using the **Run** button in the top-right side of the screen.
```

File: sources/platform/actors/running/tasks.md [hunk 3]
```diff
@@ -1,51 +1,45 @@
...
-### Identifying tasks
+### Identification
 
-To make a task easier to identify, you can give it a name, title, and description by clicking it's caption on the detail page.  A task's name should be 3-63 characters long.
+To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least 3 characters long with 63 characters limit.
 
 ### Settings
 
-You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of their detail page.
+You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of the task's detail page.
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:38]: Sharing is now under this button

CleanShot 2024-03-21 at 11 33 55@2x

File: sources/platform/actors/running/tasks.md [hunk 4]
```diff
@@ -1,51 +1,45 @@
...
 
 ![Apify task configuration](./images/tasks/create-task-configure.png)
 
-Actors' input fields will vary depending on their purpose, but they all follow the same principle: you provide an Actor with the information it needs so it can do what you want it to do.
+An Actors' input fields may vary depending on their purpose, but they all follow the same principle: _you provide an Actor with the information it needs so it can do what you want it to do._
 
 You can set run options such as timeout and [memory](./usage_and_resources.md) at the bottom of the input section.
 
-### Identifying tasks
+### Identification
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:32]: I personally prefer the old title here - sure it's longer, but I think it's clearer

File: sources/platform/actors/running/tasks.md [hunk 5]
```diff
@@ -55,3 +49,7 @@ You can also run tasks using:
 - Directly via the [Apify API](/api/v2#/reference/actor-tasks/run-collection/run-task).
 - The [JavaScript API client](/api/client/js/reference/class/TaskClient).
 - The [Python API client](/api/client/python/reference/class/TaskClient).
+
+## Share
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:53]: This is a duplicate of the `Settings` - I guess Settings can be deleted?

**Review by m-murasovs — CHANGES_REQUESTED** [review]
Cool, looks good! I guess you could also add something about integrations and issues?

File: sources/platform/actors/running/tasks.md (hunk 4 above)
Comment (TC-MO) [reply sources/platform/actors/running/tasks.md:32]: Here I tried to make sure that all the headings are in the similar style while still conveying the same information as. before

Comment (m-murasovs) [reply sources/platform/actors/running/tasks.md:32]: I see what you mean, but the old version just reads easier. Really, though, shouldn't this section just be called `Naming`?

Comment (TC-MO) [reply sources/platform/actors/running/tasks.md:32]: Yup that works, my intention here was to keep the parallel structure of headings, but `Naming` definitely achieves the same result

File: sources/platform/actors/running/tasks.md [hunk 6]
```diff
@@ -1,57 +1,56 @@
...
+An Actors' input fields may vary depending on their purpose, but they all follow the same principle: _you provide an Actor with the information it needs so it can do what you want it to do._
 
-You can set run options such as timeout and [memory](./usage_and_resources.md) at the bottom of the input section.
+You can set run options such as timeout and [memory](./usage_and_resources.md) in the **Run options** tab of the task's input configuration.
 
-### Identifying tasks
+### Naming
 
-To make a task easier to identify, you can give it a name, title, and description by clicking it's caption on the detail page.  A task's name should be 3-63 characters long.
+To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least `3` characters long with `63` characters limit.
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:35]:
```suggestion
To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least `3` characters long with a limit of `63` characters.
```

File: sources/platform/actors/running/tasks.md [hunk 7]
```diff
@@ -1,57 +1,56 @@
...
 
-### Settings
+## Run
 
-You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of their detail page.
+Once you've configured your task, you can run it using the **Start** button in the top-right side of the screen.
```
Comment (m-murasovs) [inline sources/platform/actors/running/tasks.md:39]:
```suggestion
Once you've configured your task, you can run it using the **Start** button on the top-right side of the screen.
```

**Review by m-murasovs — APPROVED** [review]
Cool! Just a couple of minor nits, but we're good to go :) pre-approving

**Pull request merged**